I created the main page of the home project.

Gavrilovph opened the pull request.

Gavrilovph committed:
Created my firs web page with HTML

Gavrilovph committed:
Update index.html

Updated links for some image tags
Small fix of h5(add a letter)

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,155 @@
+            </ul>
+    </header>
+    <main>
+        <p>01</p> <!--slide nunmber-->
+        <button><img src="/content/Rectangle (active).png" alt="active slide image"></button>
+        <button><img src="/content/Rectangle.png" alt="passive slide image"></button>
+        <button><img src="/content/Rectangle 2.png" alt="passive slide image"></button>
+        <button><img src="/content/Rectangle 3.png" alt="passive slide image"></button>
```
Кнопки можешь закинуть с список

Gavrilovph replied:
Только кнопки или «01» тоже?

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,155 @@
+        <button><img src="/content/Rectangle.png" alt="passive slide image"></button>
+        <button><img src="/content/Rectangle 2.png" alt="passive slide image"></button>
+        <button><img src="/content/Rectangle 3.png" alt="passive slide image"></button>
+        <h1>THE OOH NOO COLECTION</h1>
+        <p>Designed in Slovenia</p>
+        <button>EXPLORE NOW</button>
+        <img src="content/main image.png" alt="image of monster in main">
```
И контент баннера тоже

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,155 @@
+    </section>
+    <section>
+        <img src="/content/delivery image.jpg" alt="image of a woman">
+        <h3>DELIVERY</h3>
+        <button>Delivery</button>
+        <button>Guarantee</button>
+        <button>Credit</button>
+        <p>Some days i feel like i've been working on dlk my entire life and there are other days that i am completely shocked that four years have flown by. It's simply magical.</p>
+        <p>What started as an outlet to deal  with the traumatic loss of my husband, dlk has now become a truly happy place for me and i have you and all of the beautiful designers that we work with to thank!</p>
+        <a href="#">LAERN MORE</a>
```
Тоже оберни в список

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,155 @@
+    </header>
+    <main>
+        <p>01</p> <!--slide nunmber-->
+        <button><img src="/content/Rectangle (active).png" alt="active slide image"></button>
```
Картинки в кнопки не нужно вставлять. Внешний вид кнопкам задаётся через css

Gavrilovph replied:
Очень хотелось уже покрасивее сделать)

Gavrilovph replied:
А как тогда? Кнопки без названия делать с комментариями?

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,155 @@
+        <h1>THE OOH NOO COLECTION</h1>
+        <p>Designed in Slovenia</p>
+        <button>EXPLORE NOW</button>
+        <img src="content/main image.png" alt="image of monster in main">
```
Не оставляй пробелов в названии картинки и каждое следующее слово в названии с заглавной буквы

Gavrilovph replied:
Т.е. вот так: content/MainImage.png ???

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,155 @@
+    </section>
+    <section>
+        <div>
+            <h4>DESIGN LIFE KIDS</h4>
```
а точно h4?

Gavrilovph replied:
Саша говорил, что они должны по порядку идти.  H1-H3 уже были. А это очень похоже на заголовки, поэтому H4

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,155 @@
+        <div>
+            <h4>CONTACTS</h4>
+            <p>Some days i feel like i've been working on dlk my entire life and there are other days that i am completely shocked that four years have flown by. It's simply magical.</p>
+            <iframe src="https://yandex.ru/map-widget/v1/?ll=49.420415%2C53.507852&z=10" width="483" height="196"></iframe>
```
Круто, что вставил карту!

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,155 @@
+    </section>
+    <footer>
+            <div>
+                <h5>Navigation</h5>
```
Тут тоже самое, зачем h5?

Gavrilovph replied:
Тот же ответ, что и про h4. Саша говорил они по порядку должны идти :D

Gavrilovph replied:
Поправил тут на < P >

Gavrilovph committed:
Update main page

Изменил имя каталога, который хранит изображения, обернул некоторые блоки в div, добавил классы и элементы по БЭМ

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,211 @@
+        </a>
+        <nav class="header__navigation">
+            <ul class="header__navigation-list">
+                <li class="header__navigation-list-item"><a href="#" class="header__navigation-link">BLOG</a></li>
```
header__navigation-list-item -> header__navigation-item
Дальше элементы как бы уже находятся внутри list

Gavrilovph committed:
Add new pages

Created all pages in html and started first page in css. I'm going on ;)

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,227 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DLK</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="stylesheet" href="../yo-gavrilov-vladimir-1/styles/common.css">
```
Странный путь подключения у файла

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,227 @@
+    <title>DLK</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="stylesheet" href="../yo-gavrilov-vladimir-1/styles/common.css">
+    <link rel="stylesheet" href="./styles/normolize.css">
```
Подключение библиотек должно находиться первым в списке стилизации

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+}
+
+.container{
+    max-width: 1178px;
+    margin: auto;
+    margin-top: 50px;
+    margin-bottom: 129px;
```
max-width: 1178px; -> 1180px

Yan1993-Dev replied:
Это лишнее, не нужно указывать отступы контейнеру

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+    align-items: center;
+}
+
+.header > a:hover{
+    opacity: 50%;
+}
+
+.header > a:active{
+    opacity: 20%;
+}
```
Не задавай в %
Указывай в числовом значении
Например 0.4

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+}
+
+.header__nav-item{
+    margin: 10px;
```
Зачем со всех сторон лишке давать отступы?

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+    margin-left: 50px;
+}
+
+.header__row-list{
+    display: flex;
+    align-items: center;
+    font-size: 14px;
+    margin-left: 660px;
+}
```
header__nav блок слева, ему задай margin-right: auto;

margin-left: 660px; убери это свойство

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+    margin: 11px;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 700;
+    list-style-type: none;
```
Сделай запись на все лишки
li list-style-type: none;

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+    opacity: 20%;
+}
+
+.header__nav-item > a, .header__row-item > a{
+    text-decoration: none;
+}
```
Задай сразу на все ссылки по тегу а

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+.banner{
+    position: relative;
+    display: flex;
+    max-width: 1178px;
+    height: 411px;
+    margin: auto;
```
У тебя это и так есть в контейнере, лишний раз не задавай

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+
+.banner__img{
+    position: absolute;
+    height: 461px;
```
Зачем высоту прописываешь?
Укажи через свойство top
Оно так же принимает отрицательные значения

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+    margin-left: 31px;
+    margin-top: 10px;
+    list-style: none;
+    outline: none;
```
Не убирай outline. У тебя доступность сломается

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+}
+
+.banner__additionalInfo{
+    position: absolute;
```
Не нужно прописывать тут абсолют

Yan1993-Dev commented on styles/style.css:
```diff
@@ -0,0 +1,239 @@
+}
+
+.banner__slider-number{
+    position: relative;
```
Зачем прописывать тут релатив? 
Если нужно подвинуть, тогда используй маржин

Gavrilovph committed:
Update Style.css

Gavrilovph committed:
html fixes, css additions

Стилизовал главную страницу, страницу каталога и немного страницу toys

Gavrilovph committed:
Main page btn hover, active fix

Немного привел в порядок код, пофиксил состояния кнопки в футере.

Gavrilovph committed:
Finished Toys page css

Gavrilovph committed:
Fixed Toys page, Made styles for Blog page

Fixed input type range on Toys page.
Finished css of Blog page

Gavrilovph committed:
Final version of the project

Final version of the project

Gavrilovph committed:
Index.html Toys.html Contact-us.html and their styles fix

Added 100% background for the footer and the filter row on the Toys page.

Fixed overflow.

Fixed the inputs label clicking problem.

Fixed names off minified files.
